scripts/postinstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd /var/praekelt/goddard-captive-portal
+npm install
+
+mkdir -p /var/goddard/apps
+ln -s /var/praekelt/goddard-captive-portal /var/goddard/apps/captiveportal-redesign
```

Comment:
If not we'd need to see if the symlink already exists or not as this post-install script is run on every deploy.